src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,14 +1,40 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Min;
+import mate.academy.spring.validation.AuthValid;
+import mate.academy.spring.validation.Email;
+import org.springframework.lang.NonNull;
+
+@AuthValid(message = "passwords are not the same")
 public class UserRequestDto {
+    @Email
     private String email;
+    @NonNull
```

Comment:
```suggestion
    @NotNull
```
import javax.validation.constraints.NotNull;